libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -79,9 +79,6 @@ void AP_Logger_File::Init()
 {
     // determine and limit file backend buffersize
     uint32_t bufsize = _front._params.file_bufsize;
-    if (bufsize > 64) {
```

Comment:
Well, that's short and sweet.  Do we need to make sure this still works on PixHawk? :-)

Reply:
it's fine. We don't DMA directly from these buffers any more